_posts/2020-03-19-our-rails-upgrade-process.markdown

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: "Our Rails Upgrade Process: How to bundle update rails"
+date: 2020-03-17 10:30:00
+reviewed: 2020-03-19 10:00:00
+categories: ["rails", "upgrade"]
+author: "cleiviane"
+---
+
+We know that there are many challenges involving a Rails upgrade project. Depending on how big is your application, how old is your Rails version and how well structured your code is, it can be difficult to perform that job and keep your sanity. If you don't find a reliable and trustable process to guide you from version X to version Y, you can end-up in a nightmare.
```

Comment:
```suggestion
We know that there are many challenges involved in a Rails upgrade project. Depending on how big your application is, how old your Rails version is and how well structured your code is, it can be difficult to perform that job and keep your sanity. If you don't find a reliable and trustable process to guide you from version X to version Y, you can end-up in a nightmare.
```